db/migrate/20220224202927_add_users_ref_to_likes.rb

```diff
@@ -1,5 +1,6 @@
 class AddUsersRefToLikes < ActiveRecord::Migration[7.0]
   def change
     add_reference :likes, :user, null: false, foreign_key: true
+    add_index :user
   end
 end
```

Comment:
- [ ] While adding the like, we need to add the relationship with the like table and `post_id` because the like belongs to the post as well.